todo/src/components/Todos.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { completeTodo } from '../actions';
+
+const style = { textDecoration: 'line-through' };
+const Todos = props => {
+  return (
+    <div>
+      {props.todos.map(todo => (
+        <li
+          style={todo.completed ? style : null}
```

Comment:
Very clean use of your `style` variable here within this ternary to dynamically style your `todo` 👍